pulpcore/cli/core/task.py

```diff
@@ -125,7 +125,7 @@ def cancel(
         try:
             pulp_ctx.wait_for_task(entity)
         except Exception as e:
-            if not re.match("Task /pulp/api/v3/tasks/[-0-9a-f]*/ canceled", str(e)):
+            if not re.match(rf"Task {pulp_ctx.api_path}tasks/[-0-9a-f]*/ canceled", str(e)):
```

Comment:
Neat! I did not know you could combine r & f strings.

Reply:
Yep, i learned that too.